.devcontainer/devcontainer.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "CRC environment",
-    "image": "registry.access.redhat.com/ubi9/go-toolset:latest",
+    "image": "registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.23-openshift-4.19",
 
     "customizations": {
         "vscode": {
             "extensions": [
             ]
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
This file is first time added as part of windows but I didn't make this change except editing it in linux :)